src/db.py

```diff
@@ -173,6 +177,7 @@ def agent_get_task(self, agent_id: str) -> AgentTask:
         task_queues = [
             f"{agent_prefix}:queue-search",
             f"{agent_prefix}:queue-yara",
+            f"{agent_prefix}:queue-reload",
```

Comment:
move this to the top (order matters, when mquery is busy with yara matches it won't even try to reload config)

Reply:
Fixed